usr/sbin/pkcsslotd/slotmgr.c

```diff
@@ -491,7 +560,29 @@ static int config_parse_slot(const char *config_file,
 
         if (strcmp(c->key, "stdll") == 0 &&
             (str = confignode_getstr(c)) != NULL) {
+#if defined(_AIX)
+            /*
+             * AIX bundles libraries as ranlib archives. The objects therefore
+             * need to be loaded from within the library. Instead of modifying
+             * the parser to handle the new string format, it's easier to just
+             * build the string ourselves - this only happens at daemon init,
+             * so performance isn't a concern here.
+             */
+            char toktype[5] = {0, };
```

Comment:
move variable declaration to start of function please. There is already an #if defined(_AIX) block where you can move it to.